### ISSUE TITLE
Tutorial Deleting Your App sample message different with real test

### ISSUE THREAD

**Issue by KimGunWoo** [aws/chalice#961]
I followed tutorial steps in README document.

'Deleting Your App' sample message different with my own test.

```
$ chalice delete --stage dev
Deleting Rest API: s0xx3qzne8
Deleting function: arn:aws:lambda:us-west-2:421295790333:function:helloworld-dev
Deleting IAM role: helloworld-dev
```
There is no confirm message with my app deleting test.

Tutorial's  response message on deleting, it show confirm message on deleting the role.
```
$ chalice delete --stage dev
Deleting rest API duvw4kwyl3
Deleting lambda function helloworld-dev
Delete the role helloworld-dev? [y/N]: y    <-- this look like asking to confirm about deleting
Deleting role name helloworld-dev
```

My Test env like under:
$ chalice --version
chalice 1.6.0, python 2.7.10, darwin 17.5.0

tested on MacOS High Sierra, v10.13.4

**Comment by joguSD** [comment]
This is the expected behavior. We some fairly large changes to how deployment works, and as part of that change we no longer prompt on role deletion. We need to update this example to use the output from the latest version of Chalice.

**Comment by stealthycoin** [comment]
Fixed by #962
